plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/AccountSelector.java

```diff
@@ -107,30 +95,35 @@ public String getSelectedEmail() {
     return combo.getText();
   }
 
+  /** @exception IllegalArgumentException if there is no account logged in */
+  public String getFirstEmail() {
+    Preconditions.checkArgument(getAccountCount() > 0);
```

Comment:
If we keep the precondition, then it should be `checkState()` as we're not checking an argument of the method but the state of the object.